### ISSUE TITLE
Failing to stop stream

### ISSUE THREAD

**Issue by njam** [cargomedia/cm-janus#202]
@tomaszdurka @vogdb 
CM:
```
CM_Janus_StopStreamError: Stream stop failed in /home/example/releases/20160307142018/vendor/cargomedia/cm/library/CM/Janus/Service.php on line 104
     0. {main} /home/example/releases/20160307142018/bin/cm:0
     1. CM_Cli_CommandManager->run(CM_Cli_Arguments) /home/example/releases/20160307142018/bin/cm:18
     2. CM_Process->fork(Closure) /home/example/releases/20160307142018/vendor/cargomedia/cm/library/CM/Cli/CommandManager.php:175
     3. CM_Process->_fork(Closure, 1) /home/example/releases/20160307142018/vendor/cargomedia/cm/library/CM/Process.php:68
     4. CM_Process_ForkHandler->runAndSendWorkload() /home/example/releases/20160307142018/vendor/cargomedia/cm/library/CM/Process.php:227
     5. CM_Cli_CommandManager->{closure}(CM_Process_WorkloadResult) /home/example/releases/20160307142018/vendor/cargomedia/cm/library/CM/Process/ForkHandler.php:76
     6. CM_Cli_Command->run([], CM_InputStream_Readline, CM_OutputStream_Stream_StandardOutput, CM_OutputStream_Stream_StandardError) /home/example/releases/20160307142018/vendor/cargomedia/cm/library/CM/Cli/CommandManager.php:288
     7. call_user_func_array([], []) /home/example/releases/20160307142018/vendor/cargomedia/cm/library/CM/Cli/Command.php:27
     8. [internal function] 
     9. CM_Clockwork_Manager->start() /home/example/releases/20160307142018/vendor/cargomedia/cm/library/CM/Maintenance/Cli.php:18
    10. CM_Clockwork_Manager->runEvents() /home/example/releases/20160307142018/vendor/cargomedia/cm/library/CM/Clockwork/Manager.php:102
    11. CM_Clockwork_Manager->_runEvent(CM_Clockwork_Event) /home/example/releases/20160307142018/vendor/cargomedia/cm/library/CM/Clockwork/Manager.php:57
    12. CM_Process->fork(Closure) /home/example/releases/20160307142018/vendor/cargomedia/cm/library/CM/Clockwork/Manager.php:228
    13. CM_Process->_fork(Closure, 1493) /home/example/releases/20160307142018/vendor/cargomedia/cm/library/CM/Process.php:68
    14. CM_Process_ForkHandler->runAndSendWorkload() /home/example/releases/20160307142018/vendor/cargomedia/cm/library/CM/Process.php:227
    15. CM_Clockwork_Manager->{closure}(CM_Process_WorkloadResult) /home/example/releases/20160307142018/vendor/cargomedia/cm/library/CM/Process/ForkHandler.php:76
    16. CM_Clockwork_Event->run(DateTime) /home/example/releases/20160307142018/vendor/cargomedia/cm/library/CM/Clockwork/Manager.php:227
    17. call_user_func(Closure, DateTime) /home/example/releases/20160307142018/vendor/cargomedia/cm/library/CM/Clockwork/Event.php:54
    18. [internal function] 
    19. call_user_func_array(Closure, []) /home/example/releases/20160307142018/vendor/cargomedia/cm/library/CM/Maintenance/Cli.php:159
    20. [internal function] 
    21. CM_MediaStreams_Service->checkStreams() /home/example/releases/20160307142018/vendor/cargomedia/cm/library/CM/Maintenance/Cli.php:87
    22. CM_Janus_Service->_stopStream(CM_Model_Stream_Subscribe(304937064)) /home/example/releases/20160307142018/vendor/cargomedia/cm/library/CM/MediaStreams/Service.php:40
```

cm-janus:
```
2016-03-07 14:48:53.973 DEBUG - request /stopStream
2016-03-07 14:48:53.974 INFO - http-client request http://localhost:8300/janus/4143249887/1387878785 { janus: 'detach', transaction: 'ijfx37ip5o' }
2016-03-07 14:48:53.975 INFO - http-client response { janus: 'error',
  session_id: 4143249887,
  transaction: 'ijfx37ip5o',
  error: { code: 458, reason: 'No such session 4143249887' } }
2016-03-07 14:48:53.976 WARN - Stream stop failed [Error: http-client error: No such session 4143249887]
Error: http-client error: No such session 4143249887
    at /usr/lib/node_modules/cm-janus/lib/janus/http-client.js:36:15
    at tryCatcher (/usr/lib/node_modules/cm-janus/node_modules/request-promise/node_modules/bluebird/js/main/util.js:26:23)
    at Promise._settlePromiseFromHandler (/usr/lib/node_modules/cm-janus/node_modules/request-promise/node_modules/bluebird/js/main/promise.js:507:31)
    at Promise._settlePromiseAt (/usr/lib/node_modules/cm-janus/node_modules/request-promise/node_modules/bluebird/js/main/promise.js:581:18)
    at Promise._settlePromises (/usr/lib/node_modules/cm-janus/node_modules/request-promise/node_modules/bluebird/js/main/promise.js:697:14)
    at Async._drainQueue (/usr/lib/node_modules/cm-janus/node_modules/request-promise/node_modules/bluebird/js/main/async.js:123:16)
    at Async._drainQueues (/usr/lib/node_modules/cm-janus/node_modules/request-promise/node_modules/bluebird/js/main/async.js:133:10)
    at Immediate.Async.drainQueues [as _onImmediate] (/usr/lib/node_modules/cm-janus/node_modules/request-promise/node_modules/bluebird/js/main/async.js:15:14)
    at processImmediate [as _immediateCallback] (timers.js:367:17)
```

CM mysql - the stream in question:
```csv
"id","userId","start","allowedUntil","key","channelId"
304937064,8197249,1457360515,1457361516,"ca410815-1a22-4794-aebc-d2bfba8ffe1c",190736617
```
-> Key: **ca410815-1a22-4794-aebc-d2bfba8ffe1c**

cm-janus status:
```json
[{"id":"666879be-7044-4939-bb89-c48caeaf7f5d","channelName":"5359mpEAI9LV-KMb8ix9Gg__"},{"id":"ca410815-1a22-4794-aebc-d2bfba8ffe1c","channelName":"4AzgkXl6mXNSLZxwSnUBaw__"},{"id":"d7714915-6c03-42e3-8b5a-86ec91546a49","channelName":"p5WiRFfTAGyiMConklQQ3w__"},{"id":"04677158-bc5b-4230-8175-f09b973727cf","channelName":"p5WiRFfTAGyiMConklQQ3w__"},{"id":"6d4df731-4702-4f61-9704-17e6583dc626","channelName":"p5WiRFfTAGyiMConklQQ3w__"},{"id":"fcb71c7e-239a-4bb9-a297-f31ef939f1fd","channelName":"p5WiRFfTAGyiMConklQQ3w__"},{"id":"fafa665c-1a0d-4571-a623-ce1d3ab4251c","channelName":"p5WiRFfTAGyiMConklQQ3w__"},{"id":"09644bfc-29ae-42e5-90b7-8ebfddaa498a","channelName":"p5WiRFfTAGyiMConklQQ3w__"},{"id":"1cf4c12d-2853-4a74-b507-b0cb577a4852","channelName":"p5WiRFfTAGyiMConklQQ3w__"},{"id":"521bedfb-7acd-4084-99c8-afcb710ff478","channelName":"p5WiRFfTAGyiMConklQQ3w__"},{"id":"57e2a725-26b5-4b2d-8fcb-d5a6e961b4ef","channelName":"p5WiRFfTAGyiMConklQQ3w__"}]
```

**Comment by njam** [comment]
As I understand we have a stream, with no plugin/session?

Aleksei:
> I guess that for now we need to have this feature that stream is stopped even when there are no corresponding  session/plugin

**Comment by vogdb** [comment]
closed by #195
